content/ja/docs/reference/glossary/sig.md

```diff
@@ -0,0 +1,21 @@
+---
+title: SIG (special interest group)
+id: sig
+date: 2018-04-12
+full_link: https://github.com/kubernetes/community/blob/master/sig-list.md#master-sig-list
+short_description: >
+  大規模なKubernetesオープンソースプロジェクトにおいて、開発中の部分または側面を集合的に管理するコミュニティのメンバー
+
+aka: 
+tags:
+- community
+---
+ {{< glossary_tooltip text="Community members" term_id="member" >}} who collectively manage an ongoing piece or aspect of the larger Kubernetes open source project.
```

Comment:
This line is not localized.

Reply:
My apologies, I forgot to localize it. Will localize and commit.